apps/meteor/app/slashcommands-inviteall/server/server.ts

```diff
@@ -74,7 +74,7 @@ function inviteAll<T extends string>(type: T): SlashCommand<T>['callback'] {
 						lng,
 					}),
 				});
-			} else {
+			} else if (targetChannel) {
```

Comment:
Won't this have any change on the current logical flow?

Reply:
Removed!